### ISSUE TITLE
Theme toggle button dark mode becomes invisible with colored app bar

### ISSUE THREAD

**Issue by mangecoeur** [widgetti/solara#528]
Toggling the theme button to dark mode, the button becomes invisible because it seems to get the wrong color (not sure what colour it's picking up though)

https://github.com/widgetti/solara/assets/743508/ebedd4ab-82b9-49b7-ab03-94e37e9f0c34

With AppBar created as:

```v.AppBar(
                color="primary",
                dark=True,
                app=True,
                clipped_left=True,
                hide_on_scroll=False,
                dense=True,
            )
```

**Comment by maartenbreddels** [comment]
Yes, we have it color=primary in dark (see https://github.com/widgetti/solara/blame/11c6094f3c47528d40f8be951c1d322e454f98ea/solara/lab/components/theming.vue#L7 ).

In this example https://solara.dev/apps/scatter we change the dark setting of the toolbar, we do:
```
@solara.component
def Layout(children):
    route, routes = solara.use_route()
    dark_effective = use_dark_effective()
    return solara.AppLayout(children=children, toolbar_dark=dark_effective, color=None)
```
But I think we should not color the switch, or have it an argument. What do you think @iisakkirotko ?
